README.md

```diff
@@ -43,13 +43,7 @@ Follow these instructions to run ODK Central Frontend in development. For deploy
 
 First, run ODK Central Backend.
 
-Next, build ODK Central Frontend files for development by running `npm run dev`. The files will be outputted to `dist/`. As you update the source code, the files will be automatically rebuilt.
-
-Finally, run NGINX by changing the working directory to the root directory of the repository, then typing the following:
-
-```bash
-nginx -c "$PWD/nginx.conf" -p "$PWD"
-```
+Next, start nginx and build ODK Central Frontend files for development by running `npm run dev`. The files will be outputted to `dist/`. As you update the source code, the files will be automatically rebuilt.
 
 NGINX effectively places ODK Central Frontend and ODK Central Backend at the same origin, avoiding cross-origin requests. We specify `-p "$PWD"` so that relative paths in [`nginx.conf`](/nginx.conf) are relative to the repository root.
```

Comment:
I'm not quite sure what to do with this paragraph. The second sentence now feels like it doesn't belong in the readme. Maybe it could move to nginx.conf or Procfile or just go away? The first sentence also might be unnecessary at this point. I feel like it's nice to have some explanation of why nginx is useful, but maybe "nginx will serve the files" is enough.

Reply:
:+1: